app/Http/Requests/ChangeStatusRequest.php

```diff
@@ -24,7 +26,10 @@ public function authorize()
     public function rules()
     {
         return [
-            'status' => 'required|integer|exists:orders,status',
+            'status' => [
+                'required',
+                Rule::in([Order::PENDING, Order::ACCEPTED, Order::REJECTED]),
```

Comment:
j khổ vậy, `'required|integer|in:'.implode(',',[Order::PENDING, Order::ACCEPTED, Order::REJECTED])`